pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -172,33 +208,36 @@ func TestClusterWideMode(t *testing.T) {
 			})
 		})
 
-		// t.NewSubTest("cluster wide works with profiles").Run(func(t test.TestHelper) {
-		// 	t.Log("Check whether the cluster wide feature works with profiles")
-
-		// 	t.LogStep("Delete SMCP and SMMR")
-		// 	oc.RecreateNamespace(t, meshNamespace)
-
-		// 	t.LogStep("Create a profile with a cluster wide feature and restart OSSM operator")
-		// 	oc.CreateConfigMapFromFiles(t,
-		// 		"openshift-operators",
-		// 		"smcp-templates",
-		// 		ossm.GetProfileFile())
-		// 	podLocator := pod.MatchingSelector("name=istio-operator", "openshift-operators")
-		// 	oc.DeletePod(t, podLocator)
-		// 	oc.WaitPodReady(t, podLocator)
-
-		// 	t.LogStep("Deploy SMCP with the profile")
-		// 	oc.ApplyTemplate(t, meshNamespace, clusterWideSMCPWithProfile, ossm.DefaultSMCP())
-		// 	oc.WaitSMCPReady(t, meshNamespace, smcpName)
-
-		// 	t.LogStep("Check whether SMMR is created automatically")
-		// 	retry.UntilSuccess(t, func(t test.TestHelper) {
-		// 		oc.Get(t, meshNamespace, "servicemeshmemberroll", "default",
-		// 			assert.OutputContains("default",
-		// 				"The SMMR was created immediately after the SMCP was created",
-		// 				"The SMMR resource was not created"))
-		// 	})
-		// })
+		t.NewSubTest("cluster wide works with profiles").Run(func(t test.TestHelper) {
+			t.Log("Check whether the cluster wide feature works with profiles")
+
+			t.LogStep("Delete SMCP and SMMR")
+			oc.RecreateNamespace(t, meshNamespace)
```

Comment:
Note: deleting and recreating the namespace takes a long time because of Kiali. Whenever possible, delete the individual resources instead of the entire namespace.

Reply:
Improved this